### ISSUE TITLE
data/measurements to estimate population receptive field

### ISSUE THREAD

**Issue by yibeichan** [courtois-neuromod/cneuromod#24]
@bpinsard helloooo the _N_ th time!
Does cneuromod include any data/measurements/experiments (either fMRI or electrophysiological) that can be used to estimate the population receptive field in the visual field?

**Comment by bpinsard** [comment]
Yes, we did collect very extensive retinotopy mapping data, adapting the HCP tasks, we will soon be able to release these. 
But one of our participant couldn't tolerate the flashing of the task, so it will be missing for sub-06.
It was also analyzed with https://kendrickkay.net/analyzePRF/. @MarieStLaurent can talk more about that.

**Comment by yibeichan** [comment]
Ah, wonderful, amazing, great (whatever synonyms you can think of). @MarieStLaurent, can you provide more details and tell me when it will be released?

**Comment by yibeichan** [comment]
@bpinsard  @MarieStLaurent hello hello, when will the retinotopy mapping data be released?

**Comment by MarieStLaurent** [comment]
Hi! Sorry about that. I will need to get back to you about a clear timeline on an official data release, but we're getting close (sometime this fall). 

Do you have access to CNeuroMod data, or do you need to apply for access? The retinotopy data and its derivatives will be organized in something that looks like [this](https://github.com/courtois-neuromod/cneuromod-things/tree/dev/retinotopy).

Currently, you can access the scripts used to process the raw data and derivatives, but the data files need to be integrated into a datalad repo. The processing pipeline scripts are described [here](https://github.com/courtois-neuromod/cneuromod-things/tree/dev/retinotopy/prf/code). Files will include pRF maps, and ROIs (binary masks) identified based on those maps.

**Comment by yibeichan** [comment]
Hi, yes, I have access and have downloaded other fMRIPrepped data. Thanks for the useful links, looking forward to the data release.
